internal/db/repos/project.go

```diff
@@ -27,6 +27,13 @@ func (r *ProjectRepository) Create(ctx context.Context, project *models.Project)
 	return r.db.WithContext(ctx).Create(project).Error
 }
 
+// CreateBatch creates a batch of projects in the database
+func (r *ProjectRepository) CreateBatch(ctx context.Context, projects []*models.Project) error {
+	return r.db.WithContext(ctx).Transaction(func(tx *gorm.DB) error {
+		return tx.CreateInBatches(projects, 100).Error
```

Comment:
[nit]: `100` could be extracted as a constant

Reply:
https://github.com/celestiaorg/talis/issues/233